README.md

```diff
@@ -38,6 +38,30 @@ Another functionality of the `OperatorManifest` class is the ability to modify a
 image pull specifications identified. This is useful for performing container registry translations
 and pinning floating tags to a specific digest.
 
+## Using the CLI
+
+Once installed, this library provides the `operator-manifest` script which is a CLI wrapper for
+using the library. Notably, there are three subcommands to the script. Use
+`operator-manifest --help` for usage details.
+
+The `pin` subcommand is a "porcelain-like" command to resolve all image references found in the
+`ClusterServiceVersion` file to digests, and modify the `ClusterServiceVersion` file accordingly.
+
+The `extract` and `replace` subcommands allow users to use a custom image reference resolver. The
```

Comment:
If you're using `extract` and `replace` directly it's because you want to provide a custom process for resolving images. So you can use the command like this:
```
operator-manifest extract . | my-custom-resolver | operator-manifest replace . -
```

`my-custom-resolver` is something users can write to apply whatever custom logic they have, e.g. replacing registries.

I don't think we need something else in between:
```
operator-manifest extract . | operator-manifest TBD | my-custom-resolver | operator-manifest replace . -
```
^ That looks awkward to use.

WDYT?

Reply:
I was thinking of something like this:
```
operator-manifest extract . | operator-manifest resolve - | operator-manifest replace . -
```
Not sure if that's actually useful for anything, I was just missing one subcommand to complete the `pin` operation step by step

Reply:
Oh I see what you mean now. `resolve` could be a pretty powerful general purpose command. Let me do that in a separate PR.

Reply:
As promised: https://github.com/containerbuildsystem/operator-manifest/pull/22